src/make_android.sh

```diff
@@ -53,8 +53,81 @@ cordova plugin add cordova-plugin-whitelist@1.3.1
 cordova plugin add cordova-plugin-ionic-keyboard@2.2.0
 cordova plugin add https://github.com/manusimpson/Phonegap-Android-VolumeControl.git
 
+echo --- Reading arguments
+minsize=false
+full=false
+release=false
+sign=false
+os=false
+
+for i in $*; do 
+  if [ ${i%%=*} = "exclude-activities" ]
+  then 
+    activities=${i#*=}
+    lastActivity=${activities##*,}
+    remaining=true
+    echo -e "\n" >> exclude.android
+    cp ../sugarizer/activities.json ../sugarizer/activities.bak.json
+    while [ $remaining == true ]
+    do
+      activity=${activities%%,*}
+      activities=${activities#*,}
+      if [ ${#activity} -gt 0 ]
+      then
+        sed -i "/${activity}/d" ../sugarizer/activities.json
+        echo "activities/$activity.activity/" >> exclude.android
+      fi
+      if [ $activity = $lastActivity ]
+      then
+        remaining=false
+      fi
+    done
+    sed -i "$(( $( wc -l < ../sugarizer/activities.json) ))s/,$//" ../sugarizer/activities.json
+  elif [ ${i%%=*} = "include-activities" ]
```

Comment:
So, you can remove this `include-activities` part

Reply:
Removed in the latest commit.